Attempting to export NA loci gives bad error message.

To report a bug, fill in the information below. 
For support and feature requests, please use the discussion forum: http://discuss.hail.is/

-------------------------------------------------------------------------------------------

### Hail version:
Hail version: devel-eb1e04205793

### What you did:

Attempt to `export_vcf` with a MT where some `locus` values are `NA`.

### What went wrong (all error messages here, including the full java stack trace):

Bad error message. Didn't really tell me anything except Java Assertion Error.

[Stage 2:===========================================>       (1709 + 291) / 2000]Traceback (most recent call last):
  File "/tmp/0eccddcf-0d49-4280-8b79-c43e193b044d/vdstovcf.py", line 12, in <module>
    hail.export_vcf(myvds, 'gs://ibd-exomes/v34meta/exomes.ccdg.vcf.bgz')
  File "/home/hail/hail.zip/hail/typecheck/check.py", line 547, in wrapper
  File "/home/hail/hail.zip/hail/methods/impex.py", line 422, in export_vcf
  File "/usr/lib/spark/python/lib/py4j-0.10.4-src.zip/py4j/java_gateway.py", line 1133, in __call__
  File "/home/hail/hail.zip/hail/utils/java.py", line 196, in deco
hail.utils.java.FatalError: AssertionError: assertion failed

--- a/src/main/scala/is/hail/variant/RegionValueVariant.scala
+++ b/src/main/scala/is/hail/variant/RegionValueVariant.scala
@@ -22,8 +22,10 @@ class RegionValueVariant(rowType: TStruct) extends View {
   def setRegion(region: Region, offset: Long) {
     this.region = region
 
-    assert(rowType.isFieldDefined(region, offset, locusIdx))
-    assert(rowType.isFieldDefined(region, offset, allelesIdx))
+    if (!rowType.isFieldDefined(region, offset, locusIdx))
+      fatal(s"The row field 'locus' cannot have missing values.")
+    if (!rowType.isFieldDefined(region, offset, allelesIdx))
+      fatal(s"The row field 'alleles' cannot have missing values.")
     this.locusOffset = rowType.loadField(region, offset, locusIdx)
     this.allelesOffset = rowType.loadField(region, offset, allelesIdx)
     cachedContig = null

